src/hooks/semesters/use-semester-by-code.hook.ts

```diff
@@ -3,7 +3,7 @@ import { getSemesterByCodeApi as getSemesterByCodeApi } from '@/api/semesters/ge
 import { semesterDetailsFamily } from '@/recoil/words/semesters.state'
 
 export const useSemesterByCode = () => {
-  const handleRefresh = useRecoilCallback(
+  const onSemesterByCode = useRecoilCallback(
```

Comment:
onGetSemesterByCode

Reply:
Handled in [5ddf33e](https://github.com/ajktown/wordnote/pull/158/commits/5ddf33e05136c5e01ce02ca05840754462e6215c)